### DISCUSSION TITLE
/docs does not work on Safari beta

### DISCUSSION THREAD

**Discussion by skhaz** [fastapi/fastapi#12072] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
I have an code that works on Chrome, but not on Safari
```

### Description

* Start the server
* Open Safari
* Observe nothing happens

### Operating System

macOS

### FastAPI Version

0.112.1

### Pydantic Version

2.8.2

### Python Version

3.12.4

**Comment by YuriiMotov** [comment]
Let's close this for now to keep the discussions clean.
If you still need help, feel free to let us know.
